ui/ui-components/pages/model/[modelId]/properties.tsx

```diff
@@ -147,6 +147,7 @@ export default function Page(props) {
                       </strong>
                     </a>
                   </GrouparooLink>
+                  {rule.isPrimaryKey && " (primary)"}
```

Comment:
This is a nice touch.  Perhaps we can use a `<Badge size='small' variant='info' >primary</Badge>`?  This seems like the type of thing we badge on the source and destination already